src/components/spaces/show/index.js

```diff
@@ -136,6 +138,9 @@ export default class SpacesShow extends Component {
     this.setState({showCalculatorId: null, showCalculatorResults: false, showNewCalculatorForm: false, showEditCalculatorForm: null})
   }
 
+  showFactSidebar() { this.setState({showFactSidebar: true}) }
+  hideFactSidebar() { this.setState({showFactSidebar: false}) }
```

Comment:
You don't need this. You can just make one method to hide the entire right sidebar and put it in there, as is done with all the calculator sidebar things.